src/definitions/behaviors/api.js

```diff
@@ -1,6 +1,6 @@
 /*!
- * # Semantic UI - API
- * http://github.com/semantic-org/semantic-ui/
+ * # Fomantic UI - API
+ * http://github.com/fomantic/fomantic-ui/
```

Comment:
`Fomantic UI` should be `Fomantic-UI` and `http://github.com/fomantic/fomantic-ui/` should be `http://github.com/fomantic/Fomantic-UI` to keep consistency with other places we link to the repo.

Reply:
These changes should be for all files.

Reply:
Changed accordingly by a2b65e71322e3cab8d2e6d8084ea3a051bdb9524